source/_includes/installation/operating_system.md

```diff
@@ -205,15 +205,16 @@ Use this method only if Method 1 does not work for you.
 *Select and copy the URL or use the "copy" button that appear when you hover it.*
 
 5. Paste the URL into your browser to start the download.
-6. Select **Flash from file** and select the image you just downloaded.
+6. Extract the archive.
```

Comment:
This should not be needed.

Reply:
True, but it is.

Reply:
How is that? As in, we originally used URLs, and that one is broken in combination with `xz` files (and, unfortunately, still an open issue upstream). As a result of that issue, we have tested across multiple platforms on what the best solution would be.

The above-listed procedure is a result of that. So, wondering, if this addition is now really needed; what changed? Was this intentional? Or is it an issue? In the case of the latter, can we link it up with a known upstream issue, or do we need to create one?

Reply:
/CC @agners I think we should reconsider the release on the OS-level and maybe publish in gzip format as well; as all of these issues, as likely (at least one confirmed one) are fed by `xz`.

Reply:
Yeah unfortunately it is currently required when using the latest version of balenaEtcher :cry: 

There is a bug in balenaEtcher, see https://github.com/balena-io/etcher/issues/4064. The project appears quite stale actually :cry: . At least at one point we were start considering alternatives.

I thought at one point we recommended to use an older version (@c0ffeeca7, do you remember?)

Reply:
Ah, we changed to install from file for that reason, see: #28624.

@ludeeus so you tried to install from file and that failed too for you now? What happened exactly?

Reply:
Oh I see:

![image](https://github.com/home-assistant/home-assistant.io/assets/34061/bcea6c73-5095-4f53-bcf4-238e53409463)

Great, they broke that too. :see_no_evil: 

I'd rather prefer to move away from balenaEtcher. It will only be a question of time when gz also breaks, either from URL or file.

Reply:
I did not try it personally, but based on reports by others and the last comment (from 2 weeks ago) on the previously opened issue, made it seem like this is now an issue.
<https://github.com/balena-io/etcher/issues/4064#issuecomment-2143419306>

Reply:
Actually, gz is currently broken too:

![image](https://github.com/home-assistant/home-assistant.io/assets/34061/8d0e36a1-76f7-4ac6-98f1-d627b4bdd462)

I'd say probably best to just merge this for now? :thinking:

Reply:
> I'd say probably best to just merge this for now?

I agree, better to have working documentation.

Reply:
Agree, thanks for the additional context here @agners 👍 

../Frenck